requirements.txt

```diff
@@ -0,0 +1,189 @@
+acstools==3.2.0
```

Comment:
That's a long list of dependencies. I supposed you've generated with `pip freeze` or some `condo` command. Are all of those necessary?
One problem I see with this list is that is pinned to specific versions. That makes it easier to test it and deliver software that is guaranteed to work. On the other hand, this app will be hard to coexist with other apps and it will be hard to automatically get automatic dependency updates.

Reply:
Hmm, good point and food for thought... I've removed the `requirements.txt` for and replaced it with requirements in `setup.cfg` for now.

Reply:
In general, I put the application requirements in `setup.cfg` and the development requirements (`pytest`, `coverage`) in a `dev_requirements.txt` file.

Reply:
Awesome, 171d9315 has set up a `dev_requirements.txt`.